src/core/janeway_global_settings.py

```diff
@@ -515,3 +516,9 @@ def __len__(self):
 # A potentially dangerous feature, this allows superusers to hijack and control a user's account.
 HIJACK_USERS_ENABLED = False
 HIJACK_LOGIN_REDIRECT_URL = '/manager/'
+
+CORE_FILETEXT_MODEL = "core.FileText"
+if os.environ.get("DB_VENDOR") == "postgres":
```

Comment:
Note to self - we should check the vendor in the connection rather than the env variable.